packages/web-workers/src/messenger/index.ts

```diff
@@ -1,2 +1,7 @@
 export {createWorkerMessenger} from './worker';
-export {createIframeWorkerMessenger} from './iframe';
+export {
+  createNullOriginIframeMessenger as createIframeWorkerMessenger,
+  createNullOriginIframeMessenger,
+  createRemoteOriginIframeMessenger,
+  IFRAME_RUN_IDENTIFIER,
```

Comment:
Decided to export the const as folks could use that to generate iFrame host files whenever they update remote-ui.

Reply:
I don't think you should export this as a public API, so you can safely remove it from here and the `iframe/index.ts` file